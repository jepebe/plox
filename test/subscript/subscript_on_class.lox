class A {
    init() {
        this.a = 1;
    }

    __get__ (index) {
        return '[' + index + ']';
    }

}

print A[0];
