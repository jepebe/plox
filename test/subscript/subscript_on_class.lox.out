[38;5;1m[RuntimeError at line 12] Subscript not supported.[0m
